apps/portals/src/portal-components/challengeportal/ChallengeParticipantGoogleMap.tsx

```diff
@@ -1,11 +1,10 @@
 import { Box } from '@mui/material'
 import * as React from 'react'
-import { SynapseQueries, Map } from 'synapse-react-client'
+import { AppUtils, SynapseQueries, Map } from 'synapse-react-client'
 
 const ChallengeParticipantGoogleMap = () => {
-  const urlSearchParams = new URLSearchParams(window.location.search)
-  const entityId = urlSearchParams.get('id') ?? ''
-  const { data: challenge } = SynapseQueries.useGetEntityChallenge(entityId)
+  const projectId = AppUtils.useQuerySearchParam('id')
+  const { data: challenge } = SynapseQueries.useGetEntityChallenge(projectId)
   if (challenge && challenge.participantTeamId) {
```

Comment:
Use a new useQuerySearchParam hook (defined later in the PR) to grab a single parameter from the URL.

Reply:
Also found that for the GenUI team, there was no geolocation file, which caused the app to crash (because it was attempting to parse json from the html response).  So I fixed this call as well (to only attempt to parse the json if the status code is 200)